content-service/src/main/java/io/meeds/news/utils/NewsUtils.java

```diff
@@ -94,6 +94,11 @@ public enum NewsObjectType {
     DRAFT, LATEST_DRAFT, ARTICLE;
   }
 
+  public enum NewsUpdateType {
+    CONTENT, SCHEDULE, POSTING_AND_PUBLISHING
+  }
+
```

Comment:
```suggestion
```

Reply:
it space between the enum class and the method for the readability of the code !